doc: doc and test URLSearchParams discrepancy (#33236)
nodejs/node: doc/api/url.md

* {URLSearchParams}

Gets the [`URLSearchParams`][] object representing the query parameters of the
URL. This property is read-only but the `URLSearchParams` object it provides
can be used to mutate the URL instance; to replace the entirety of query
parameters of the URL, use the [`url.search`][] setter. See
[`URLSearchParams`][] documentation for details.

> Care must be taken when using `.searchParams` to modify the `URL` because,

Reviewer: Remove one of the instances of passive voice here:

```suggestion
Use care when using `.searchParams` to modify the `URL` because,
```

> per the WHATWG specification, the `URLSearchParams` object uses slightly

Reviewer: Eliminate unnecessary adverb:

```suggestion
per the WHATWG specification, the `URLSearchParams` object uses
```

> different rules to determine which characters should be percent-encoded. For

Reviewer: ```suggestion
different rules to determine which characters to percent-encode. For
```
